.circleci/config.yml

```diff
@@ -132,28 +160,90 @@ jobs:
       - run: docker build -f Dockerfile.tests -t marquez-airflow-base .
       - run: ./tests/integration/docker/up.sh
 
-  release:
+  release-java:
+    <<: *machine
+    steps:
+      - checkout
+      - run: ./.circleci/get-jdk11.sh
+      - run: ./gradlew --no-daemon -x test :api:release -Prelease.useAutomaticVersion=true -Prelease.releaseVersion=$CIRCLE_TAG
+      - run: ./gradlew --no-daemon -x test :clients:java:release -Prelease.useAutomaticVersion=true -Prelease.releaseVersion=$CIRCLE_TAG
+      - run: ./gradlew --no-daemon -x test :integrations:spark:release -Prelease.useAutomaticVersion=true -Prelease.releaseVersion=$CIRCLE_TAG
+      - run: ./gradlew publish
+
+
+  release-docker:
     <<: *machine
     steps:
       - checkout
+      - run: ./.circleci/get-jdk11.sh
+      - run: ./.circleci/replace-gradle-version.sh $CIRCLE_TAG gradle.properties api/gradle.properties
       - run: ./docker/login.sh
       - run: ./docker/build-and-push.sh $CIRCLE_TAG
 
+  release-python:
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: ls -la /tmp/workspace
+      - run: pip install wheel twine
+      - run: python -m twine upload --non-interactive --repository pypi /tmp/workspace/*
+
 workflows:
   marquez:
     jobs:
       - build-api
       - build-web
```

Comment:
_Minor_: You can rename `build-web` to `test-web` to better represent the function of the job.